.github/workflows/build-docker-images.yml

```diff
@@ -307,3 +307,41 @@ jobs:
   #           REF=main
   #         push: true
   #         tags: huggingface/transformers-pytorch-deepspeed-amd-gpu-push-ci
+
+  latest-quantization-torch-docker:
+    name: "Latest Pytorch + Quantization [dev]"
+     # Push CI doesn't need this image
+    if: inputs.image_postfix != '-push-ci'
+    runs-on: ubuntu-22.04
```

Comment:
let's from now on move this to `[intel-cpu, 8-cpu, ci]` (our own VM, bigger RAM and disk)

Reply:
done !